modules/rtm/DESCRIPTION

```diff
@@ -1,37 +1,29 @@
-Package: PEcAn.rtm
+Package: PEcAnRTM
 Type: Package
 Title: PEcAn functions used for radiative transfer modeling
 Version: 1.4.1
 Date: 2015-01-22
-Author: David LeBauer, Mike Dietze, Xiaohui Feng, Dan Wang, Mike Dietze, Carl
-    Davidson, Rob Kooper, Shawn Serbin
-Maintainer: David LeBauer <dlebauer@illinois.edu>
+Author: 
+    Alexey Shiklomanov, David LeBauer, Mike Dietze,
```

Comment:
Ok! I pulled these from the original package and added my name, but I can reduce this list to people I know contributed.